interfaces/Config/templates/config_switches.tmpl

```diff
@@ -321,11 +321,7 @@
                     <div class="field-pair">
```

Comment:
It always used to be in Specials. Moved it to Switches with the changes in 1.1.0, but turns out  indexers actually prefer to send their own custom rating URL instead of just a host and having to use the endpoint that was once hardcoded by OZnzb.